.github/workflows/build-with-bratiska-cli.yml

```diff
@@ -71,6 +71,14 @@ jobs:
       - name: Directory check
         run: pwd
 
+      - name: Cache Docker layers
+        uses: actions/cache@v4
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-docker-${{ hashFiles('**/Dockerfile') }}
+          restore-keys: |
+            ${{ runner.os }}-docker-
```

Comment:
Doing docker cache like this is a bit unfortunate as this has a lot of implications. Most important one is a security issue that you are using [public GitHub action storage](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/caching-dependencies-to-speed-up-workflows#about-caching-workflow-dependencies) (warning is what you want to read) to cache layers which are basically plain text files.

A better way would be to use some other [build cache backend](https://docs.docker.com/build/cache/backends/) like registry and point it to our harbor, or if that wouldn't work to use S3, through their official GitHub action.